src/models/Token.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const { sequelize } = require('../utils/db.js');
+const { DataTypes } = require('sequelize');
+const { User } = require('./User.js');
+
+const Token = sequelize.define('Token', {
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Token.belongsTo(User);
```

Comment:
Associations are being declared directly. It's better to encapsulate them in a class method for better code organization.

Reply:
Example?